packages/asset-swapper/docs/reference.mdx

```diff
@@ -0,0 +1,2872 @@
+
+*Defined in [order_providers/standard_relayer_api_order_provider.ts:18](https://github.com/0xProject/0x-monorepo/blob/bfbc78c95/packages/asset-swapper/src/order_providers/standard_relayer_api_order_provider.ts#L18)*
+
+___
+
+##  networkId
+
+• **networkId**: *number*
+
+*Defined in [order_providers/standard_relayer_api_order_provider.ts:19](https://github.com/0xProject/0x-monorepo/blob/bfbc78c95/packages/asset-swapper/src/order_providers/standard_relayer_api_order_provider.ts#L19)*
+
+### Methods
+
+##  getAvailableMakerAssetDatasAsync
+
+▸ **getAvailableMakerAssetDatasAsync**(`takerAssetData`: string): *`Promise<string[]>`*
+
+*Defined in [order_providers/standard_relayer_api_order_provider.ts:92](https://github.com/0xProject/0x-monorepo/blob/bfbc78c95/packages/asset-swapper/src/order_providers/standard_relayer_api_order_provider.ts#L92)*
+
+Given a taker asset data string, return all available paired maker asset data strings.
+
+**Parameters:**
+
+Name | Type | Description |
+------ | ------ | ------ |
+`takerAssetData` | string | A string representing the taker asset data. |
+
+**Returns:** *`Promise<string[]>`*
+
+An array of asset data strings that can be purchased using takerAssetData.
+
+___
+
+##  getAvailableTakerAssetDatasAsync
+
+▸ **getAvailableTakerAssetDatasAsync**(`makerAssetData`: string): *`Promise<string[]>`*
+
+*Defined in [order_providers/standard_relayer_api_order_provider.ts:120](https://github.com/0xProject/0x-monorepo/blob/bfbc78c95/packages/asset-swapper/src/order_providers/standard_relayer_api_order_provider.ts#L120)*
+
+Given a maker asset data string, return all availabled paired taker asset data strings.
+
+**Parameters:**
+
+Name | Type | Description |
+------ | ------ | ------ |
+`makerAssetData` | string | A string representing the maker asset data. |
+
+**Returns:** *`Promise<string[]>`*
+
+An array of asset data strings that can be used to purchased makerAssetData.
+
+___
+
+##  getOrdersAsync
+
+▸ **getOrdersAsync**(`orderProviderRequest`: [OrderProviderRequest](#interface-orderproviderrequest)): *`Promise<OrderProviderResponse>`*
+
+*Defined in [order_providers/standard_relayer_api_order_provider.ts:68](https://github.com/0xProject/0x-monorepo/blob/bfbc78c95/packages/asset-swapper/src/order_providers/standard_relayer_api_order_provider.ts#L68)*
+
+Given an object that conforms to OrderProviderRequest, return the corresponding OrderProviderResponse that satisfies the request.
+
+**Parameters:**
+
+Name | Type | Description |
+------ | ------ | ------ |
+`orderProviderRequest` | [OrderProviderRequest](#interface-orderproviderrequest) | An instance of OrderProviderRequest. See type for more information. |
+
+**Returns:** *`Promise<OrderProviderResponse>`*
+
+An instance of OrderProviderResponse. See type for more information.
+
+<hr />
+
+# Class: ExchangeSwapQuoteConsumer
+
+### Hierarchy
+
+* **ExchangeSwapQuoteConsumer**
+
+### Implements
+
+* [SwapQuoteConsumerBase](#exchangesmartcontractparams)*›
```

Comment:
This type anchor link seems off:
`[SwapQuoteConsumerBase](#exchangesmartcontractparams)`

Also a weird hanging `*>` here.